packages/core/base/src/models/types.ts

```diff
@@ -113,17 +125,34 @@ export interface NFTCollectionViewProps extends CommonProps {
     wallets: Wallet[];
 }
 
-export interface NFT {
-    address: string;
-    chain: string;
-    tokenId?: string;
+export interface CardanoNFT {
```

Comment:
interface AbstractNFT = {
    chain: BlockchainType
} 

EVMNFT extends AbstractNFT?

Reply:
that woudn't fit here, since I want dedicated chain types for each...

Reply:
hhmmm...

Ok agreed that we might need some extra validation for javascript users. Lets do it in next iteration.